docker/conda_build.dockerfile

```diff
@@ -0,0 +1,98 @@
+ARG ubuntu_version=22.04
+
+FROM ubuntu:${ubuntu_version} AS openmc
```

Comment:
Let's call this stage `pyne-deps`.  `openmc` made sense when we had other stages before it, built now we only build this stage as the dependencies